vyxal/elements.py

```diff
@@ -4268,9 +4281,9 @@ def zero_matrix(lhs, ctx):
     """
     mat = []
     temp = 0
-    for index in iterable(lhs, ctx=ctx):
+    for ind in iterable(lhs, ctx=ctx):
```

Comment:
Deepsource was complaining about redefining `index` here so I renamed it to `ind`

Reply:
that's understandable - name masking is a bad thing